backend/spec/system/front_end/question_views_spec.rb

```diff
@@ -2,14 +2,15 @@
 
 RSpec.describe 'Question', type: :system do
   let!(:user) { create(:user_student) }
-  let!(:question) { create(:multiple_choice_question) }
 
   describe '#show' do
     before do
       sign_in user
       visit question_path question
     end
 
```

Comment:
Usually, we put `let` above `before`

Reply:
Yes, but then it always visits the page where multiple question is. 
And the test with single and free text question are not passing.